bin/sdmerge

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
Maybe not?

```
~/Development/quasar-analytics/quasar/master (master) λ which python2
~/Development/quasar-analytics/quasar/master (master) λ /usr/bin/env python2
env: python2: No such file or directory
~/Development/quasar-analytics/quasar/master (master) λ 
```

Reply:
Ok, seems like they can do their PEPs but noone cares :stuck_out_tongue: 
In that case, to not spend too much time on it it is enough if anyone with python 3 as default would simply adjust this line himself.